changelogs/unreleased/3443-dependency-extras.yml

```diff
@@ -0,0 +1,4 @@
+description: Fix installing extras of module dependencies
+issue-nr: 3443
+change-type: patch
+destination-branches: [master, iso5]
```

Comment:
* Let's add this to the bugfix section of the changelog.
* I think this bug also exists on ISO4. Should we merge it there as well?